Firestore/Example/Tests/Local/FSTLocalStoreTests.mm

```diff
@@ -67,12 +69,25 @@
 using firebase::firestore::remote::WatchTargetChangeState;
 using firebase::firestore::util::Status;
 
+using testutil::Array;
+using testutil::Doc;
+using testutil::Key;
+using testutil::Map;
 using testutil::Query;
+using testutil::Vector;
 
-static NSArray<FSTDocument *> *docMapToArray(const DocumentMap &docs) {
-  NSMutableArray<FSTDocument *> *result = [NSMutableArray array];
+static std::vector<MaybeDocument> DocMapToArray(const MaybeDocumentMap &docs) {
```

Comment:
Optional nit: prefer putting helper function in an anonymous namespace to declaring them `static` (that namespace could also contain the using declarations above).

Reply:
It's recommended in [Tips of the Week](https://abseil.io/tips/119#Summary):
> _ Declare namespace aliases and using-declarations inside the innermost namespace, whether named or anonymous. (Do not add an anonymous namespace just for this purpose.)_

Note: I do _not_ mean to change it in this PR now, it's not worth the trouble.